### PR TITLE
Added specific errors for tenant creation

### PR DESCRIPTION
**Pull request by Ninjeneer**

### CODE REVIEW

File: src/app/pages/tenants/tenant/tenant.component.ts [hunk 1]
```diff
@@ -313,7 +313,13 @@ export class TenantComponent implements OnInit {
       }
     }, (error) => {
       this.spinnerService.hide();
-      Utils.handleHttpError(error, this.router, this.messageService, this.centralServerService, 'tenants.create_error');
+      if (error.status === HTTPError.TENANT_NAME_ALREADY_EXIST) {
```
Comment (LucasBrazi06) [inline src/app/pages/tenants/tenant/tenant.component.ts:316]: You can remove this check on the name.

Comment (LucasBrazi06) [inline src/app/pages/tenants/tenant/tenant.component.ts:316]: Use a switch/case

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]

File: src/app/pages/tenants/tenant/tenant.component.ts (hunk 1 above)
Comment (Ninjeneer) [reply src/app/pages/tenants/tenant/tenant.component.ts]: Removed

Comment (Ninjeneer) [reply src/app/pages/tenants/tenant/tenant.component.ts]: Done

**Review by LucasBrazi06 — APPROVED** [review]
GUT!

**Pull request merged**